src/app.js

```diff
@@ -40,16 +43,33 @@ var run = function () {
   var self = this
   this.event = new EventManager()
   var storage = new Storage()
+  var files = new Files(storage)
   var config = new Config(storage)
+  var currentFile
+
+  function packageFiles () {
+    return files.list().map(function (path) { return { content: files.get(path) } })
```

Comment:
Yes, but we can add a simple line `.filter(function (path) { if (!files.isReadOnly(path) return path })` to exclude readonly.